BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/TransactionDocs/Models/TransactionDoc.swift

```diff
@@ -11,3 +11,11 @@ struct TransactionDoc {
     let fileName: String
     let type: TransactionDocType
 }
+
+extension TransactionDoc {
+    static func createMockDocuments() -> [TransactionDoc] {
+        let doc1 = TransactionDoc(documentId: "doc1", fileName: "filename1", type: .document)
+        let doc2 = TransactionDoc(documentId: "doc2", fileName: "filename2", type: .document)
+        return [doc1, doc2]
+    }
+}
```

Comment:
[clarification] Do we still need this since we have `TransactionDoc+Mock`file?

Reply:
oh, I accidentally commited it, should be deleted, thank you!